jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java

```diff
@@ -24,131 +24,123 @@
+        assertThat(illegalArgumentException).hasMessage("No parent profile 'i-dont-exist' defined");
     }
 
     @Test
-    public void mergeProfiles() throws Exception {
+    void mergeProfiles() throws Exception {
         Profile profile = ProfileUtil.findProfile("merge-1", getProfileDir());
-        assertFalse(profile.getEnricherConfig().use("jkube-project-label"));
-        assertTrue(profile.getEnricherConfig().use("jkube-image"));
+        assertThat(profile.getEnricherConfig().use("jkube-project-label")).isFalse();
+        assertThat(profile.getEnricherConfig().use("jkube-image")).isTrue();
     }
 
     @Test
-    public void blendProfiles() throws Exception {
+    void blendProfiles() throws Exception {
 
         ProcessorConfig origConfig = new ProcessorConfig(Arrays.asList("i1", "i2"), Collections.singleton("spring.swarm"), null);
         ProcessorConfig mergeConfig = ProfileUtil.blendProfileWithConfiguration(ProfileUtil.ENRICHER_CONFIG,
             "simple",
             Collections.singletonList(getProfileDir()),
             origConfig);
-        assertTrue(mergeConfig.use("base"));
-        assertTrue(mergeConfig.use("i1"));
-        assertEquals("http://jolokia.org", mergeConfig.getConfig().get("base").get("url"));
-
+        assertThat(mergeConfig.use("base")).isTrue();
+        assertThat(mergeConfig.use("i1")).isTrue();
+        assertThat(mergeConfig.getConfig().get("base")).containsEntry("url", "http://jolokia.org");
 
         mergeConfig = ProfileUtil.blendProfileWithConfiguration(ProfileUtil.GENERATOR_CONFIG,
             "simple",
             Collections.singletonList(getProfileDir()),
             origConfig);
-        assertTrue(mergeConfig.use("i2"));
-        assertFalse(mergeConfig.use("spring.swarm"));
-
+        assertThat(mergeConfig.use("i2")).isTrue();
+        assertThat(mergeConfig.use("spring.swarm")).isFalse();
     }
 
     @Test
-    public void shouldExtend() throws Exception {
+    void shouldExtend() throws Exception {
         Profile aProfile = ProfileUtil.findProfile("minimal", getProfileDir());
 
-        assertEquals("minimal", aProfile.getName());
-        assertEquals("simple", aProfile.getParentProfile());
+        assertThat(aProfile.getName()).isEqualTo("minimal");
+        assertThat(aProfile.getParentProfile()).isEqualTo("simple");
 
-        assertTrue(aProfile.getEnricherConfig().use("jkube-name"));
-        assertTrue(aProfile.getEnricherConfig().use("default.service"));
-        assertTrue(aProfile.getGeneratorConfig().use("spring.swarm"));
-        assertFalse(aProfile.getGeneratorConfig().use("java.app"));
+        assertThat(aProfile.getEnricherConfig().use("jkube-name")).isTrue();
+        assertThat(aProfile.getEnricherConfig().use("default.service")).isTrue();
+        assertThat(aProfile.getGeneratorConfig().use("spring.swarm")).isTrue();
+        assertThat(aProfile.getGeneratorConfig().use("java.app")).isFalse();
```

Comment:
It would be nice if we could combined all of the assertions into a single assertThat() statement.